tests/test_server_basic.py

```diff
@@ -716,6 +716,50 @@ def test_list_file(dispatcher_live_fixture):
     assert job_id == calculated_job_id
 
 
+def test_value_range(dispatcher_long_living_fixture):
```

Comment:
this is useful, but I do not really understand why is it part of this PR?

Reply:
It came out from our discussion and then I decided to put in this PR, just to avoid to create another one only for a single test

Reply:
ok, I did not check what test in the plugin breaks, if this is analogous to it, it's nice

Reply:
A range check for E1 value, so I reproduced here a similar one, after the change I applied.